docs/src/main/asciidoc/logging.adoc

```diff
@@ -9,7 +9,7 @@ include::_attributes.adoc[]
 :categories: core,getting-started,observability
 :diataxis-type: reference
 
-Read about the use of logging API in Quarkus, configuring logging output, and using logging adapters to unify the output from other logging APIs.
+Read about the use of the JBoss Logging API in Quarkus, configuring logging output, and using logging adapters to unify the output from other logging APIs.
```

Comment:
A few lines below we mention that one can use one of many APIs... so I don't think this change makes sense

Reply:
Ok. Good point. I will patch it like this:
`Read about the use of logging APIs in Quarkus, configuring logging output, and using logging adapters for unified output.`
Thank you, @yrodiere 
Cheers!